azure/packages/external-controller/src/app.ts

```diff
@@ -162,8 +162,7 @@ async function start(): Promise<void> {
 		initialContainers: [
 			{
 				container,
-				containerId: id,
-				containerNickname: "Dice Roller Container",
+				id: "Dice Roller Container",
```

Comment:
Are we giving clients the control over setting `id` when they are initializing the `devtools`?

Reply:
Yeah. The idea is to make the identifier arbitrary, since we don't really care if they give us the object ID of the container, or a document ID, or just an arbitrary string. Our system strictly needs a way to differentiate containers that are registered with it. So this PR sort of merges the old `containerId` and `containerNickname` props into a single prop.

Reply:
I went ahead and renamed it to just "containerName" to doubly avoid confusion and updated documentation. Let me know what you think.

Reply:
Actually, the more I think about it, the more I like the idea of calling it `containerKey` - I think it better implies how the value is used by the system. I'm going to go with that path and see what we think.